spec/grape/api_spec.rb

```diff
@@ -4738,4 +4738,26 @@ def uniqe_id_route
       expect(last_response.body).to eq('unknown params_builder: unknown')
     end
   end
+
+  describe '.lint!' do
+    let(:app) do
+      Class.new(described_class) do
+        lint!
+        get '/' do
+          status 42
+        end
+      end
+    end
+
+    around do |example|
+      Grape.config.lint = false
+      example.run
+      Grape.config.lint = true
```

Comment:
I think this needs an `ensure` because `example.run` can raise. NBD.

Reply:
Makes total sense. Thank you for pointing that out